<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainOperate</class>
 <widget class="QMainWindow" name="MainOperate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainOperate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#mainFrame{
	background-color:#fff;
}</string>
   </property>
   <widget class="QFrame" name="menuFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>200</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#menuFrame QPushButton{
	color:black;
	padding-left:15px;
	text-align:left;
	background-color:rgba(0,0,0,0);
}
#menuFrame QPushButton:hover{
	background-color:rgba(0,0,0,0.2);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="item1">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>壁纸设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="item2">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>任务栏透明</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="item3">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="item4">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="item5">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="item6">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>无</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>561</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
